content/account_management/rbac/role_api.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Role API
```

Comment:
Should this be its own page? Or should we create a section for it in https://docs.datadoghq.com/api ?

Reply:
i think we should ultimately add it to the API docs, but since it's in private beta right now, maybe we should keep it separate? It might change a lot before it goes GA in Q1.

Reply:
APIs should be on the API page